### DISCUSSION TITLE
Empty Block Text

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [Z3d0X/filament-fabricator#64] [Q&A] [answered]
I have block without any parmeters how can add text in center without any input?
Like this?
<img width="790" alt="Screenshot 2023-05-10 at 6 40 27 PM" src="https://github.com/Z3d0X/filament-fabricator/assets/47450090/f1ac9576-724f-4a93-b404-3219eebdacbc">

**Comment by Z3d0X** [comment, marked as answer]
Use a [Placeholder](https://filamentphp.com/docs/2.x/forms/layout#placeholder) or a [View Field](https://filamentphp.com/docs/2.x/forms/layout#view)

**Reply by mstfkhazaal** [reply]
Thanks
